src/pages/workspace/categories/ImportedCategoriesPage.tsx

```diff
@@ -0,0 +1,147 @@
+            return;
+        }
+
+        const columns = Object.values(spreadsheet?.columns ?? {});
+        const categoriesNamesColumn = columns.findIndex((column) => column === CONST.CSV_IMPORT_COLUMNS.NAME);
+        const categoriesGLCodeColumn = columns.findIndex((column) => column === CONST.CSV_IMPORT_COLUMNS.GL_CODE);
+        const categoriesEnabledColumn = columns.findIndex((column) => column === CONST.CSV_IMPORT_COLUMNS.ENABLED);
+        const categoriesNames = spreadsheet?.data[categoriesNamesColumn].map((name) => name);
+        const categoriesEnabled = categoriesEnabledColumn !== -1 ? spreadsheet?.data[categoriesEnabledColumn].map((enabled) => enabled) : [];
+        const categoriesGLCode = categoriesGLCodeColumn !== -1 ? spreadsheet?.data[categoriesGLCodeColumn].map((glCode) => glCode) : [];
+        const categories = categoriesNames?.slice(containsHeader ? 1 : 0).map((name, index) => ({
+            name,
+            enabled: categoriesEnabledColumn !== -1 ? categoriesEnabled?.[containsHeader ? index + 1 : index] === 'true' : true,
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            'GL Code': categoriesGLCodeColumn !== -1 ? categoriesGLCode?.[containsHeader ? index + 1 : index] : '',
+        }));
+
+        if (categories) {
+            setIsImportingCategories(true);
+            importPolicyCategories(policyID, categories);
+        }
+    }, [validate, spreadsheet, containsHeader, policyID]);
+
+    const spreadsheetColumns = spreadsheet?.data;
+    if (!spreadsheetColumns) {
+        return;
+    }
+
+    return (
+        <ScreenWrapper
+            testID={ImportedCategoriesPage.displayName}
+            includeSafeAreaPaddingBottom
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.categories.importCategories')}
+                onBackButtonPress={() => Navigation.goBack(ROUTES.WORKSPACE_CATEGORIES_IMPORT.getRoute(policyID))}
+            />
+            <ImportSpreadsheetColumns
+                spreadsheetColumns={spreadsheetColumns}
+                columnNames={columnNames}
+                importFunction={importCategories}
+                errors={isValidationEnabled ? validate() : undefined}
+                columnRoles={columnRoles}
+                isButtonLoading={isImportingCategories}
+                headerText={translate('workspace.categories.importedCategoriesMessage')}
+                learnMoreLink={CONST.IMPORT_SPREADSHEET.CATEGORIES_ARTICLE_LINK}
+            />
+
+            <ConfirmModal
+                isVisible={spreadsheet?.shouldFinalModalBeOpened}
+                title={spreadsheet?.importFinalModal?.title ?? ''}
+                prompt={spreadsheet?.importFinalModal?.prompt ?? ''}
+                onConfirm={() => {
+                    setIsImportingCategories(false);
+                    closeImportPage();
+                    Navigation.navigate(ROUTES.WORKSPACE_CATEGORIES.getRoute(policyID));
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/56936, when we close import page and modal, we're using `navigate`, which is causing a bug. This adds `ImportedCategoriesPage` to the navigation stack, so when we go back from `Categories`, it takes us to `ImportedCategoriesPage` again. Instead, we could use `goBack` here. More context on this comment: https://github.com/Expensify/App/issues/56936#issuecomment-2662960972